src/App.js

```diff
@@ -11,14 +19,73 @@ class App extends React.Component {
       cardDeck: makeShuffledDeck(),
       // currCards holds the cards from the current round
       currCards: [],
+      currComputerCards: [],
+      roundWinner: "",
+      score: [0, 0],
+      overallWinner: "",
+      roundsLeft: 26,
     };
   }
 
   dealCards = () => {
     // this.state.cardDeck.pop() modifies this.state.cardDeck array
-    const newCurrCards = [this.state.cardDeck.pop(), this.state.cardDeck.pop()];
+    const newCurrCards = [this.state.cardDeck.pop()];
+    const newComputerCurrCards = [this.state.cardDeck.pop()];
+    this.setState(
+      {
+        currCards: newCurrCards,
+        currComputerCards: newComputerCurrCards,
+        roundsLeft: this.state.roundsLeft - 1,
+      },
+      () => this.determineWinner()
```

Comment:
```suggestion
      this.determineWinner
```

I think you should be able to just pass the function itself like so